xclim/indices/_threshold.py

```diff
@@ -812,6 +1151,8 @@ def tropical_nights(
       Minimum daily temperature [℃] or [K]
     thresh : str
       Threshold temperature on which to base evaluation [℃] or [K]. Default: '20 degC'.
+    greater_or_equal_to : bool
```

Comment:
Really ? What's the rationale for this ?

Reply:
The PCC's formula differs from what we have already. While these cases are few and far between with simulated data, they are much more likely to happen with observations that have much less precision. I agree that this is really ugly. It's probably better to split this into a new function.

Reply:
Or use the same function with 19.9999 as the threshold. 
Are we mostly using > or >= elsewhere ?

Reply:
We tend to use a mix of both. It would be good to have a standard for most indices, but changing existing ones would be a major breaking change. Worth considering in a new PR.

Reply:
Ok, so I'm -1 on adding `greater_or_equal` type arguments to indices. 
Indeed, this PR is big enough already.

Reply:
One thing I've seen in some of the indices is a function called `utils.threshold_count`. The way it's written could technically allow for us to specify the type of operator for a given indice, e.g. 

```f = indices.tropical_nights(tas_dataset, thresh="20 degC", operator="gteq | gt | > | >=")```

Could this be something to consider for the refactor?

Reply:
@huard thoughts on this? @aulemahal and I have discussed this issue and have been considering proposing this approach in another issue. Is this worth taking the time to look at?

Reply:
Can we leave this out of this PR for now and have another discussion on this. I'm not keen to add this level of detail to our API.